docs/learn-parathreads.md

```diff
@@ -4,7 +4,7 @@ title: Parathreads
 sidebar_label: Parathreads
 ---
 
-Parathreads are an idea for parachains to temporarily participate in Polkadot security without needing to lease a dedicated parachain slot. This is done through economically sharing the scarce resource of a _parachain slot_ among a number of competing resources (parathreads). Chains that otherwise would not be able to acquire a full parachain slot are enabled to still participate in Polkadot's shared security -- albeit with an associated fee per block. It also offers a graceful off-ramp to parachains which no longer require a dedicated parachain slot, but would like to continue using the relay chain.
+Parathreads are an idea for parachains to temporarily participate (on a block by block basis) in Polkadot security without needing to lease a dedicated parachain slot. This is done through economically sharing the scarce resource of a _parachain slot_ among a number of competing resources (parathreads). Chains that otherwise would not be able to acquire a full parachain slot, or do not find it economically sensible to do so, are enabled to still participate in Polkadot's shared security -- albeit with an associated fee per block. It also offers a graceful off-ramp to parachains which no longer require a dedicated parachain slot, but would like to continue using the relay chain.
```

Comment:
Well, compared to e.g. moving from Tron to EOS, it's seamless (or at least a very small seam).  It's just not instantaneous.  We could add some more info about what is involved in moving from one to the other, though

Reply:
That's a good idea, seamlessness is relative :)

Reply:
Since this was really a "clean-up" PR, and we will want to go into detail a bit on this, I am going to spin it off into a new issue - put it in issue #66